test/support/workarea/paypal_setup.rb

```diff
@@ -0,0 +1,49 @@
+module Workarea
+  module PaypalSetup
+    extend ActiveSupport::Concern
+
+    included do
+      setup :set_paypal_credentials
+      teardown :reset_paypal_credentials
+    end
+
+    def set_paypal_credentials
+      @env_paypal_client_id = ENV['WORKAREA_PAYPAL_CLIENT_ID']
+      @env_paypal_client_secret = ENV['WORKAREA_PAYPAL_CLIENT_SECRET']
+
+      # If you need to re-record any VCR cassettes, you will need to replace
+      # these will real credentials, then update the OAuth request's
+      # basic auth header to use these credentials, base64 encoded 'ID:SECRET',
+      # in the recorded request to prevent committing real credentials.
```

Comment:
This is helpful, maybe it'd be worth a section in the README? Either way, could you expand a little bit or maybe write in a list of steps?

Reply:
Sure. Probably going to make a separate PR for the README unless i finish before this gets merged.